SoftLayer/CLI/core.py

```diff
@@ -5,12 +5,14 @@
 
     :license: MIT, see LICENSE for more details.
 """
+import json
 import logging
 import os
 import sys
 import time
 import traceback
 import types
+import urllib3
```

Comment:
use `requests`,  which contains its own json stuff, so you can remove the json import as well.

And add this  constant:
`PROG_NAME = "slcli (SoftLayer Command-line)"`